_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
+
+~~~YAML
+rootpath:
+  CMIP3: [~/cmip3_inputpath1, ~/cmip3_inputpath2]
+  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+  CMIP6: [~/cmip6_inputpath1, ~/cmip6_inputpath2]
+  OBS: ~/obs_inputpath
+  OBS6: ~/obs6_inputpath
+  obs4mips: ~/obs4mips_inputpath
+  ana4mips: ~/ana4mips_inputpath
+  native6:  ~/native6_inputpath
+  RAWOBS: ~/rawobs_inputpath
+  default: ~/default_inputpath
 ~~~
+
+In this lesson, we will work with data from [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/).
+We add the root path of the folder where data is available.
+
+~~~YAML
   rootpath:
   ...
-    CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2, ~/escience-academy/test_data]
+    CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2, ~/esmvaltool_tutorial/data]
 ~~~
-{: .source}
 
-## Auxiliary data directory (used for some additional datasets)
-  auxiliary_data_dir: ~/auxiliary_data
+> ## Setting the correct rootpath
+>
+> * To get the data (or its correct rootpath), check instruction in [Setup](https://esmvalgroup.github.io/tutorial/setup.html).
+> * For more information about setting the rootpath, you can visit ESMValTool [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/datafinder.html).
+{: .callout}
 
-The ``auxiliary_data_dir`` setting is the path to place any required
-additional auxiliary data files. This method was necessary because certain
-Python toolkits such as cartopy will attempt to download data files at run
-time, typically geographic data files such as coastlines or land surface maps.
-This can fail if the machine does not have access to the wider internet. This
-location allows us to tell cartopy (and other similar tools) where to find the
-files if they can not be downloaded at runtime. To reiterate, this setting is
-not for model or observational datasets, rather it is for data files used in
-plotting such as coastline descriptions and so on.
+## Directory structure for the data from different projects
 
+Input data can be from various models, observations and reanalysis data that adhere to the [CF/CMOR standard](https://cmor.llnl.gov/). To set a directory, you can use one of the values of ``default``, ``BADC``, ``DKRZ``, ``ETHZ``, .... Let's use ``default`` in our example:
 
-## Output settings
+~~~YAML
+drs:
+  CMIP5: default
+~~~
+
+> ## Available drs
+>
+> For more information about directories, you can visit ESMValTool [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/config.html#developer-configuration-file).
+{: .callout}
+
+## Number of parallel tasks
 
-.. code-block:: yaml
+This option enables you to perform parallel processing. You can choose the number of tasks in parallel as 1/2/3/4/... or you can set it to ``null`` that tells ESMValTool to use the number of available CPUs:
```

Comment:
I would rephrase to "... or you can set it to ``null``. That tells ESMValTool...".